src/images/icons/rotate_90.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="512px" height="512px" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
we need to format that here, take a look at other icons - it should have the same format

Reply:
you can take a look at `CONTRIBUTING.md` => `### Adding icons process`